docs/source/markdown/podman-kube-play.1.md

```diff
@@ -145,6 +145,12 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+#### **--detach**, **-d**
+
+Return after the pods have been started. The default is **true**.
```

Comment:
How about
```
       Detached mode: run Pods and containers in the. The default is true.

       At  any time you can run `podman pod ps` the other shell to view a list of
       the running pods and containers. 

       When  attached  in the tty mode, you can kill the pods and containers by pressing
       Ctrl-C or receiving any other interrupt signals.
```

Reply:
nit change to @rhatdan suggesting "the other shell" -> "in the other shell"